src/microbe_stage/FloatingChunk.cs

```diff
@@ -221,6 +233,12 @@ public override void _Ready()
             chunkMesh = graphicsNode.GetNode<MeshInstance>(ModelNodePath);
         }
 
+        if (!string.IsNullOrEmpty(AnimationPath))
+        {
+            animation = graphicsNode.GetNode<AnimationPlayer>(AnimationPath);
+            animation.GetAnimation(animation.CurrentAnimation).Loop = true;
```

Comment:
For cilia chunks, I'd actually think it should not loop, but this is the default behaviour now, so no need to specifically code that now.
But if you actually have to break up the glb file, you should be able to edit and force the animation to autoplay in looping mode, so this like could be commented out with a TODO comment to make type specific code to stop animations of specific chunks. https://github.com/Revolutionary-Games/Thrive/issues/3357